### ISSUE TITLE
Reproducing paper results

### ISSUE THREAD

**Issue by tomergi** [hongzimao/pensieve#79]
I'm trying to reproduce the results of the paper. I crated cooked traces using the same datasets used in the paper, and used a 80/20 split for training and testing. It seems like MPC performs better than both the pretrained model and models I tried to train. I added the fix in issue #20 and I use the following schedule for the entropy weight:
5 until epoch 20000
4 until epoch 30000
3 until epoch 40000
2 until epoch 50000
1 until epoch 60000
0.5 until epoch 70000
0.2 until epoch 80000
0.1 until epoch 90000
0.01 until epoch 100000 where the training ends.
I get these results:
![problem_with_pensieve_training](https://user-images.githubusercontent.com/5486408/59979807-26ae6400-95f5-11e9-8ff1-a7b573c0b665.png)

What am I doing wrong? Is the total number of epochs correct?

**Comment by ShagadatovNurlan** [comment]
Hi, have you succeeded in results reproduction? And have changed initial code of mps.py, bb.py or dp.cc to get proper plots?

**Comment by swimming16** [comment]
Are you using synthetic data as cooked traces？

**Comment by hongzimao** [comment]
No, cooked traces are generated from a broadband and a cellular network trace datasets. More details in the paper section 5.1, "a broadband dataset provided by the FCC [10] and a 3G/HSDPA mobile dataset collected in Norway [37]."
